### ISSUE TITLE
custom-segmented-control not found in Android

### ISSUE THREAD

**Issue by infaz** [wix/react-native-custom-segmented-control#10]
Working fine in iOS, giving error in Android

<img width="348" alt="screen shot 2018-10-17 at 4 54 57 pm" src="https://user-images.githubusercontent.com/6937825/47083160-6f737900-d22d-11e8-917f-ce62bd3bd0ef.png">

**Comment by shivsuthanm-odhub** [comment]
Yes I'm also facing issues in android. Is this library only works for iOS or Android too ??

**Comment by theRealGupta** [comment]
is there any solution to this problem? I am also waiting for this solution.
